build.rs

```diff
@@ -40,5 +48,60 @@ fn main() {
         println!("cargo:rustc-link-lib=static=chck-xdg");
         println!("cargo:rustc-link-lib=static=wlc-protos");
         println!("cargo:rustc-link-lib=static=wlc");
+    } else if let Err(s) = find() {
+        let _ = writeln!(io::stderr(), "{}", s);
+        process::exit(1);
+    }
+}
+
+fn find() -> Result<(), Error> {
+
+    let package_name = "wlc";
+    let shared_libs = ["wlc"];
+
+    // https://github.com/way-cooler/rust-wlc#rust-wlc currently
+    // (i.e. commit 97e0d5816fe7ec97dae20196d1e25c550f60f08f) suggests
+    // version 0.0.7.
+    let version = if cfg!(feature = "0.0.8") {
```

Comment:
The docs to `Config::new()` state 

> Creates a new set of configuration options which are all initially set to "blank".

That is all I know.  Specifically, I don't know how options set to "blank" are treated.

https://docs.rs/pkg-config/0.3.9/pkg_config/ states in its second example:

> Find the system library named foo, with no version requirement (not recommended):

I wouldn't pin code on any other code without having inspected the other code first.